### DISCUSSION TITLE
Introduction to Rx.NET 2nd Edition (2024) book is available for FREE

### DISCUSSION THREAD

**Discussion by HowardvanRooijen** [dotnet/reactive#2078] [Announcements]
*TLDR; we've written a FREE book about Reactive Extensions for .NET called Introduction to Rx.NET 2nd Edition (2024), which is available in [PDF](https://endjincdn.blob.core.windows.net/assets/ebooks/introduction-to-rx-dotnet/introduction-to-rx-dotnet-2nd-edition.pdf), [EPUB](https://endjincdn.blob.core.windows.net/assets/ebooks/introduction-to-rx-dotnet/introduction-to-rx-dotnet-2nd-edition.epub), [Online](https://introtorx.com), and on [GitHub](https://github.com/dotnet/reactive/blob/main/Rx.NET/Documentation/IntroToRx/00_Foreword.md).*

![Introduction to Rx.NET 2nd Edition (2024)](https://res.cloudinary.com/endjin/image/upload/v1705050242/assets/images/blog-graphics/free-introduction-to-rx-dotnet-2nd-edition-book-2023-now-available.png)

Many thanks to [Lee Campbell](https://github.com/LeeCampbell) for donating the contents of his 2010 book to the project, and to @idg10 for spending a large part of 2023H2 modernising it.

More info here: https://endjin.com/blog/2024/01/introduction-to-rx-dotnet-2nd-edition-book-available-for-free

**Comment by Bomret** [comment]
Congratulations! This is huge!

**Reply by HowardvanRooijen** [reply]
Thank you!

**Comment by thinker1990** [comment]
Good Job!
